src/routes.rs

```diff
@@ -301,4 +330,32 @@ mod tests {
             }
         }));
     }
+
+    #[tokio::test]
+    async fn given_grue_id_when_getting_her_data_then_return_data() {
+        let server = given_test_server(None);
+        let body_grue = GrueRequest {
+            grue_id: VALID_GRUE_ID,
+            number_of_merchandise: ANY_NUMBER_OF_MERCHANDISE,
+        };
+
+        let mut path = String::from(GRUE_PATH);
```

Comment:
Did you know there exists the [`Path` structure](https://doc.rust-lang.org/std/path/struct.Path.html) to join paths automatically ? :)